test/project_types/script/layers/domain/script_json_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+require "project_types/script/test_helper"
+
+describe Script::Layers::Domain::ScriptJson do
+  let(:content) do
+    {
+      "version" => "1",
+      "title" => "Some Title",
+      "description" => "Some Description",
+      "configurationUi" => true,
+      "configuration" => {},
+    }
+  end
+
+  subject { Script::Layers::Domain::ScriptJson.new(content: content) }
+
+  describe "#initialize" do
+    it "constructs a ScriptJson" do
+      assert_equal("1", subject.version)
+      assert_equal("Some Title", subject.title)
+      assert_equal("Some Description", subject.description)
+      assert(subject.configuration_ui)
```

Comment:
Should we test the case where `configurationUi` is set to false? Just to ensure we're not accidentally always using the default value.